pkg/strategy/grid2/strategy.go

```diff
@@ -1766,6 +1775,8 @@ func (s *Strategy) getWriteContext(fallbackCtxList ...context.Context) context.C
 func (s *Strategy) Run(ctx context.Context, _ bbgo.OrderExecutor, session *bbgo.ExchangeSession) error {
 	instanceID := s.InstanceID()
 
+	s.filledOrderIDSet = map[uint64]struct{}{}
```

Comment:
maybe use types.OrderMap? or types.SyncOrderMap for thread safefy?

Reply:
Okay, I'll take a look.